README.md

```diff
@@ -10,13 +10,15 @@ It is heavily inspired by Brunch.
 
 See [parched-example-app](https://github.com/raisedmedia/parched-example-app), or [parched-tasks-webapp](https://github.com/raisedmedia/parched-tasks-webapp) even. The following will be based on them.
 
+A list of ready made plugins can be found at [parched-plugins](https://github.com/BraveNewWorldDev/parched-plugins)
```

Comment:
This would be a better link for that: https://www.npmjs.com/browse/keyword/parched